### DISCUSSION TITLE
☓マルチ範囲→○多重範囲（multirangeの訳を統一）

### DISCUSSION THREAD

**Discussion by KenichiroTanaka** [pgsql-jp/jpug-doc#2438] [NGとする語]
## 重要度: （中）（高/中/低から選んでください）
中

## 問題点
multirangeの訳が
A.マルチ範囲(release-14.sgml)
B.多重範囲(func.sgmlとcatalogs.sgml)
で揺れています。

## 解決方法
googleで検索した限りはどちらが一般的かを確認できませんでした。
すでにより多くcommitされているBで良いように思いますが、ご意見ありましたらお知らせください。

**Comment by koizumistr** [comment]
#2200 にも書いたように、揺れていることを問題視しているだけで、どちらか一方が良い、という強い意見は持っていないです。

> すでにより多くcommitされているBで良い

というのは一理あると思います。（なので、多数決で決めますとか言われたらBに投票しますが、、、）

**Comment by noborus** [comment]
私もB.多重範囲 で良いと思ってますが、馴染みがないので問題ないのかわかりません。
@harukat さんどうでしょうか？

**Comment by KenichiroTanaka** [comment]
B.多重範囲が多数でしたのでこちらに合わせました。
斉藤さん、PRによる対応ありがとうございました。
